posthog/api/annotation.py

```diff
@@ -23,6 +23,8 @@ class Meta:
             "date_marker",
             "creation_type",
             "dashboard_item",
+            "dashboard",
```

Comment:
We conventionally serialize IDs as `foo_id`, and only say `foo` when serializing the entity as an object/dict: 
```suggestion
            "dashboard_id",
```